build/images/scripts/install_cni

```diff
@@ -5,7 +5,7 @@ set -euo pipefail
 # Todo: check version and continue installation only for a newer version
 
 # Install Antrea configuration file
-install -m 644 /etc/antrea/antrea-cni.conf /host/etc/cni/net.d/10-antrea.conf
+install -m 644 /etc/antrea/antrea-cni.conflist /host/etc/cni/net.d/10-antrea.conflist
```

Comment:
I am not sure you have addressed Quan's comment already: https://github.com/vmware-tanzu/antrea/pull/557#issuecomment-605764276 ?

If you haven't found another way, I suggest just adding `rm -f /host/etc/cni/net.d/10-antrea.conf` here with a comment explaining why this is necessary.

Reply:
i have not yet addressed his comments.. this was something i missed in my previous commit so decided to push it before i address Quans comment..

Reply:
done